INSTALL.md

```diff
@@ -119,6 +119,15 @@ export PKG_CONFIG_PATH="$PKG_CONFIG_PATH:$(brew --prefix)/opt/libpq/lib/pkgconfi
 export PKG_CONFIG_PATH="$PKG_CONFIG_PATH:$(brew --prefix)/opt/openssl@3/lib/pkgconfig"
 export PATH="$(brew --prefix bison)/bin:$PATH"
 ```
+- Install `clang-format-12` 
+> Note: `brew` does not contain a cask for `clang-format-12`, and other versions of `clang-format`  have different formatting behavior.
+
+To install `clang-format-12`, run the one-liner below. It will download [clang+llvm-12.0.0-x86_64-darwin.tar.xz](https://github.com/llvm/llvm-project/releases/download/llvmorg-12.0.0/clang+llvm-12.0.0-x86_64-apple-darwin.tar.xz), unarchive the downloaded file, and move the contents of `bin/clang-format` to `/usr/local/bin`. 
+
+```curl -OL https://github.com/llvm/llvm-project/releases/download/llvmorg-12.0.0/clang+llvm-12.0.0-x86_64-apple-darwin.tar.xz && tar -xf clang+llvm-12.0.0-x86_64-apple-darwin.tar.xz && cd clang+llvm-12.0.0-x86_64-apple-darwin && sudo cp -R * bin/clang-format /usr/local/bin```
```

Comment:
Nit: I don't think we need this to be a one liner, and it would be a lot more readable if this was in a code block instead of inlined. We also don't need the cd:
```
curl -OL https://github.com/llvm/llvm-project/releases/download/llvmorg-12.0.0/clang+llvm-12.0.0-x86_64-apple-darwin.tar.xz 
tar -xf clang+llvm-12.0.0-x86_64-apple-darwin.tar.xz 
sudo cp -R clang+llvm-12.0.0-x86_64-apple-darwin/* bin/clang-format /usr/local/bin
```